README.md

```diff
@@ -27,255 +20,154 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 | Java / Groovy | Yes                | Yes                |
 | C++           | No                 | Yes                |
 | JavaScript    | No                 | Yes                |
-| gRPC          | -                  | Yes                |
+| Go            | No                 | Yes                |
+| R             | No                 | Yes                |
+
+## Installation
+
+Deephaven Community Core can be used from both server- and client-side APIs. Server-side APIs allow users to connect directly to the Deephaven server and execute commands, write queries, and much more. Client-side APIs allow users to connect to a Deephaven server through a client application.
+
+See the following documentation links for installation instructions and more:
+
+- Python
+  - [Run from Docker](https://deephaven.io/core/docs/tutorials/quickstart/)
```

Comment:
I'm sad about this page (and the groovy one).

I think we should be advertising this as our quickstart:

```sh
docker run --rm --name deephaven -p 10000:10000 ghcr.io/deephaven/server:0.27.1
```

a bit quicker than needing to pull down a compose file.